### PR TITLE
A styles to project page and workflow_description

### LINKED ISSUE #371: Allow description header and text
Allowing a description header and a description text field would allow projects to make effective landing pages without having to open up the customization conversation (#295). The description header would appear in large bold text (as a header1 or something) and the description text would be plain text that is the same text rendered elsewhere on the site. In the example below,  "Help map the wildebeest migration!" would be the header, and the rest would be the description that appears on the landing page and wherever else it's needed. 

image

### PR DESCRIPTION
**Pull request by edpaget**
Closes #371

### CODE REVIEW

**Comment by edpaget** [comment]
Dependent on zooniverse/Panoptes#1014

**Comment by edpaget** [comment]
Good to go now.

File: css/project-page.styl [hunk 1]
```diff
@@ -59,13 +59,58 @@
...
+      font-size: 15px
+      color: #5a5a5a
+      text-align: center
+      max-width: 800px
+      margin: 0 auto
+
+    p
+      color: #989898
+      font-size: 1em
+      line-height: 1em
```
Comment (aweiksnar) [inline css/project-page.styl]: These `p`s look a bit cramped with multiple lines. I'd say bump this line-height up a bit or remove this line and then it's good to go :+1:

Comment (edpaget) [inline css/project-page.styl]: I made it 1.4em let know how that looks

Comment (aweiksnar) [inline css/project-page.styl]: LGTM!

File: app/pages/lab/project-details.cjsx [hunk 2]
```diff
@@ -185,6 +185,15 @@ module.exports = React.createClass
             <small className="form-help">Add a brief introduction to get people interested in your project. This will display on your landing page. Note this field renders markdown (<insert link to best markdown tutorial>), so you can add formatting.</small>
           </p>
 
+          <p>
+            <AutoSave resource={@props.project}>
+              <span className="form-label">Workflow Description</span>
+              <br />
+              <textarea className="standard-input full" name="description" value={@props.project.workflow_description} onChange={handleInputChange.bind @props.project} />
```
Comment (brian-c) [inline app/pages/lab/project-details.cjsx]: Unless I'm misunderstanding this field, it looks like the `name` attribute is wrong here.

Comment (edpaget) [inline app/pages/lab/project-details.cjsx]: Yes I'll change that right away.

**Pull request merged**